apps/dashr-wind-steaming/README.md

```diff
@@ -0,0 +1,44 @@
+# Dash Wind Streaming
+
+## About this app
+
+This app queries a SQL database every second and uses the data to update the wind speed diagram and the wind direction diagram. 
+The wind speed values are then binned in real time to generate the wind histogram plot.
+
+Original repo: [plotly/dash-wind-streaming](https://github.com/plotly/dash-wind-streaming)
+
+
+## How to run this app
+
+(The following instructions apply to Posix/bash. Windows users should check
```

Comment:
Is there a more user-friendly way to write this than "... apply to Posix/bash"? 😬 

Maybe something like "... instructions are for Linux and OS X users."?

I don't believe you _have_ to be using `bash` for these notes to be helpful.